src/Bicep.Core/Syntax/DeclarationSyntax.cs

```diff
@@ -0,0 +1,19 @@
+﻿// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+using System.Collections.Generic;
+using System.Collections.Immutable;
+
+namespace Bicep.Core.Syntax
+{
+    public abstract class StatementSyntax : SyntaxBase

```

Comment:
>StatementSyntax [](start = 26, length = 15)

Should we call it DeclarationSyntax to match the file name?

Reply:
Ah good catch...too bad my ReSharper license expired which should have caught this. I renamed the file to `StatementSyntax` though. It's kind of weird if we call it `DeclarationSyntax` while it has nothing to do with the `IDeclaration` interface.